changelog.md

```diff
@@ -13,8 +13,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Setup global application design.
 * Setup authentication from `leto-modelizer`.
 * Setup Users page
-* Setup Libraries page with:
-  * Table to display all libraries
 * Setup Roles page with:
   * Table to display all roles
+* Setup Libraries page with:
+  * Table to display all libraries
+  * Add action to redirect on library page.
+* Setup Library page with:
+  * Display all information of a library.
```

Comment:
`.` or not `.` ? this is the question

Reply:
![image](https://github.com/ditrit/leto-modelizer-admin/assets/6179956/baf6c6eb-d07e-4c3e-b3d1-4ee648ab7ab9)